src/components/MessageComposer.react.js

```diff
@@ -1,11 +1,11 @@
 
 import * as Actions from '../actions';
-import React from 'react';
+import React,{Component} from 'react';
 import { PropTypes } from 'prop-types';
 
 let ENTER_KEY_CODE = 13;
 
-class MessageComposer extends React.Component {
+class MessageComposer extends Component {
```

Comment:
Thanks for the work, but you should consider other best practices as this change, like changing components to lightweight stateless components where we don't require state :)